src/components/ScmRepositoryDetailPage/ScmRepositoryDetailPage.tsx

```diff
@@ -22,16 +24,27 @@ import * as scmRepositoryApi from 'services/scmRepositoryApi';
 import { generateScmRepositoryName } from 'utils/entityNameGenerators';
 import { generatePageTitle } from 'utils/titleHelper';
 
-export const ScmRepositoryDetailPage = () => {
+interface IScmRepositoryDetailPageProps {
+  componentId?: string;
+}
+
+export const ScmRepositoryDetailPage = ({ componentId = 's2' }: IScmRepositoryDetailPageProps) => {
```

Comment:
`s1`

Reply:
we already have `ScmRepositoriesPage = ({ componentId = 's1' }`,  I would like to name it differently.

Reply:
It does not matter as long as the pages are different, but okay. The only important thing is that component IDs are distinct and used consistently on one page. Other than that, the name is inconsequential.